mex/backend/rules/helpers.py

```diff
@@ -45,10 +45,10 @@ def transform_raw_rules_to_rule_set_response(
 
     if len(set(stem_types)) != 1:
         msg = "inconsistent rule item stem types"
-        raise MExError(msg)
+        raise InconsistentGraphError(msg)
     if len(set(stable_target_ids)) != 1:
         msg = "inconsistent rule item stableTargetIds"
-        raise MExError(msg)
+        raise InconsistentGraphError(msg)
```

Comment:
What do you think about adding the stableTargetIds to the error messages to ease debugging?